### ISSUE TITLE
[MESSAGES] No validation when we enter message without a title

### ISSUE THREAD

**Issue by sdgun** [appirio-tech/connect-app#226]
Description
-----------------------------------------------------------------
[MESSAGES] No validation when we enter message without a title

Steps to Reproduce
-----------------------------------------------------------------

- Open the sites 
- Login as a valid user (pat_monahan)
- In the Projects
- Click on a Project (https://connectv2.topcoder-dev.com/projects/336)
- Click Messages
- Enter spaces to the title and add a descriptipn
- Click POST

Expected Result(s)
-----------------------------------------------------------------
- Must validate it

Actual Result
-----------------------------------------------------------------
- No validation when we enter message without a title

Environment
-----------------------------------------------------------------
- **Device(s):** Laptop
- **Browser(s):** Chrome 53.0.2785.116 m, FF 49.0.1
- **Operating System:** Windows 7 Pro 64bit
- **User Accounts:** pat_monahan

Image/Video/JS Log (If not attached here, Please check the comment section) 
-----------------------------------------------------------------

**Comment by sdgun** [comment]
![2016-09-29_5-15-09](https://cloud.githubusercontent.com/assets/4592712/18936288/ba539748-8603-11e6-9c0b-87c929f55d5f.png)

**Comment by fnisen** [comment]
Yes, valid bug. In fact, messages without a title are failing for me when I try to post, with no feedback.

**Comment by vikasrohit** [comment]
@fnisen @vic-appirio How should we show the feedback (error) when either title or the content is empty?
